engine/src/main/java/org/destinationsol/game/sound/OggMusicManager.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Why are we messing around with sound files? (There are 5 sounds files here that shouldn't be)